src/components/Anime/AnimeFetch.js

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { AIRING_URL, SEARCH_URL, TOP_URL } from "../../constants/MyAnimeList";
+
+export default AnimeFetch = async ({
+    type,
+    page,
+    search,
+    onSuccesfulFetch,
+    abortController = new AbortController(),
+}) => {
+    let url;
+
+    switch (type) {
```

Comment:
Instead of using a string to check the cases, it is better to use a pre-declared constant since a string is more prone to error and harder to manage. For example, you can define 
`
const fetchType = {
  SEARCH: 0,
  AIRING: 1,
  TOP: 2
}
`